src/app/providers/Theme.tsx

```diff
@@ -29,6 +29,32 @@ const ThemeProviderWithToggle = ({ children }: PropsWithChildren) => {
     [mode],
   );
 
+  useEffect(() => {
```

Comment:
Boah! It's look like vanilla JS O_o

Reply:
kinda =) MUI has integrate function for CSS but it's broken and they are not going to fix it